.gitignore

```diff
@@ -23,4 +23,4 @@ $RECYCLE.BIN/
 Thumbs.db
 UserInterfaceState.xcuserstate
 .env
-src/components.d.ts
+src/**/components.d.ts
```

Comment:
Nitpick, `components.d.ts` is always generated directly under `src`, no?

Reply:
it's added to wherever the config sirDir property is set to, which was changed to src/components to prevent some of our other source code from being added to the dist/

Reply:
Ah, good to know. Thanks!